doc/api/worker_threads.md

```diff
@@ -1957,6 +1957,18 @@ this matches its values.
 
 If the worker has stopped, the return value is an empty object.
 
+### `worker.startCpuProfile(name)`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+* name: {string}
+* Returns: {Promise}
+
+Starting a CPU profile with the given `name` for the worker thread. The profile can be stopped
+with [`worker.stopCpuProfile(name)`][].
```

Comment:
Similar to the other PR, I'd much prefer an API like

```js
const profile = await worker.startCpuProfile(name);
// do stuff
await profile.stop();
```

or with explicit resource management

```js
{
  await using profile = await worker.startCpuProfile(name);
  // do stuff
}
```

Reply:
Done !